README.md

```diff
@@ -175,6 +177,8 @@ index.addObjects(contactsJSON, function(err, content) {
 
 ### Search
 
+> for this use: [**search only** API key](https://www.algolia.com/doc/guides/security/api-keys/#search-only-api-key) in client instantiation
```

Comment:
Just for consistency we can move this section below the explanation, like you did with the others.